roles/openshift_logging/defaults/main.yml

```diff
@@ -13,10 +13,26 @@ openshift_logging_curator_run_minute: 0
 openshift_logging_curator_run_timezone: UTC
 openshift_logging_curator_script_log_level: INFO
 openshift_logging_curator_log_level: ERROR
+openshift_logging_curator_cpu_limit: 100m
+openshift_logging_curator_memory_limit: null
```

Comment:
Are templates check for 'none' which is the same as null when defined in the yaml.  Undefined and none are not the same which means we have to check for both in the template if that's the behavior we wish to support.  I wish they were essentially one in the same, but i think its cleaner to just define them as null and check for 'none'